src/test/kotlin/net/emdal/tank/CreateQueryTest.kt

```diff
@@ -0,0 +1,80 @@
+package net.emdal.tank
+
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+
+class CreateQueryTest {
+
+  @Test
+  fun `create empty node`() {
+    val query = Graph().create {
+      node()
+    }.query
+
+    query eq """
+      CREATE ()
+    """.trimIndent()
+  }
+
+  @Test
+  internal fun `create node with alias`() {
+    val query = Graph().create {
+      node("n")
+    }.query
+
+    query eq """
+      CREATE (n)
+    """.trimIndent()
+  }
+
+
+  // TODO: Discuss multiple node creation syntax
+  @Disabled
+  @Test
+  internal fun `create multiple nodes with different aliases`() {
+    val query = Graph().create {
+      node("n")
+      node("m")
+    }.query
+
+    query eq """
+      CREATE (n), (m)
+    """.trimIndent()
+  }
+
+  @Test
+  internal fun `create a node with a label`() {
+    val query = Graph().create {
+      node("n", Recipe)
+    }.query
+
+    query eq """
+      CREATE (n:Recipe)
+    """.trimIndent()
+  }
+
+
+  // TODO: Discuss multiple labels
```

Comment:
Really good news! This is valid syntax for Kotlin:
```
fun node(alias:String, vararg:Node, lambda: () -> Unit) {
}
```
Looks like we can have both vararg and last argument lambdas in the same function

Reply:
Starting on a solution for multiple labels